test/parallel/test-buffer.js

```diff
@@ -1438,3 +1438,14 @@ assert.equal(Buffer.prototype.parent, undefined);
 assert.equal(Buffer.prototype.offset, undefined);
 assert.equal(SlowBuffer.prototype.parent, undefined);
 assert.equal(SlowBuffer.prototype.offset, undefined);
+
+{
+  // Test that large negative Buffer length inputs don't affect the pool offset.
```

Comment:
That was @Fishrock123 in https://github.com/nodejs/node/pull/7051#discussion_r65076107